### DISCUSSION TITLE
Does Velero support snapshot volume on NetApp CVS?

### DISCUSSION THREAD

**Discussion by ccna2ccda** [vmware-tanzu/velero#8299] [Community support Q&A]
Currently, we are using Kopia integration/FSB for the mounted NetApp CVS volumes in the pods.  Is it supported to use snapshot volume instead of pod volume (FSB)? Will it gain some performance benefit? Thanks.

**Comment by kaovilai** [comment]
check netapp csi docs if it has snapshot capability. IIUC [it should](https://github.com/NetApp/trident#:~:text=management%20technology%20for-,snapshots,-%2C%20backups%2C%20replication%2C%20and).
